rpcs3/Emu/FS/vfsDirBase.h

```diff
@@ -62,14 +62,12 @@ class vfsDirBase
 
 	public:
 		iterator(vfsDirBase* parent)
-			: parent(parent)
-			, data(parent->First())
+			: data(parent->First())
 		{
 		}
 
 		iterator(const DirEntryInfo* data)
-			: parent(parent)
```

Comment:
shouldn't this be  this->parent(parent)? or is parent not a member of iterator? (can't see in this diff)